src/odemis/driver/xt_client.py

```diff
@@ -0,0 +1,344 @@
+        available_channels = self.client.list_available_channels()
+        return available_channels
+
+    def move_stage(self, position, rel=False):
+        """
+        Move the stage the given position in meter. This is non-blocking.
+
+        Parameters
+        ----------
+        position: dict(string->float)
+            Absolute or relative position to move the stage to per axes in m. Axes are 'x' and 'y'.
+        rel: boolean
+            If True the staged is moved relative to the current position of the stage, by the distance specified in
+            position. If false the stage is moved to the absolute position.
+        """
+        self.client.move_stage(position, rel)
+
+    def stage_is_moving(self):
+        """Returns True if the stage is moving and False if the stage is not moving."""
+        return self.client.stage_is_moving()
+
+    def stop_stage_movement(self):
+        """Stop the movement of the stage."""
+        self.client.stop_stage_movement()
+
+    def get_stage_position(self):
+        """
+        Returns
+        -------
+        position: dict
+            The current position of the stage.
+        """
+        return self.client.get_stage_position()
+
+    def stage_info(self):
+        """Returns the unit and range of the stage position."""
+        return self.client.stage_info()
+
+    def acquire_image(self, channel_name='electron1'):
+        """
+        Acquire an image observed via the currently set channel. Note: the channel needs to be stopped before an image
+        can be acquired. To acquire multiple consecutive images the channel needs to be started and stopped. This
+        causes the acquisition speed to be approximately 1 fps.
+
+        Returns
+        -------
+        image: numpy array
+            The acquired image.
+        """
+        return self.client.acquire_image(unicode(channel_name))
+
+    def set_scan_mode(self, mode):
+        """
+        Set the scan mode.
+        Parameters
+        ----------
+        mode: str
+            Name of desired scan mode, one of: unknown, external, full_frame, spot, or line.
+        """
+        self.client.set_scan_mode(unicode(mode))
+
+    def set_selected_area(self, start_position, size):
+        """
+        Specify a selected area in the scan field area.
+
+        Parameters
+        ----------
+        start_position: (tuple of int)
+            (x, y) of where the area starts in pixel, (0,0) is at the top left (checked for Apreo).
+        size: (tuple of int)
+            (width, height) of the size in pixel.
+        """
+        self.client.set_selected_area(start_position, size)
+
+    def get_selected_area(self):
+        """
+        Returns the current selected area in x, y, width, height in pixels.
+        If selected area is not active it returns the stored selected area.
+        """
+        x, y, width, height = self.client.get_selected_area()
+        return x, y, width, height
```

Comment:
directly return?

Reply:
I think in this specific case an indirect return is more readable. because then you directly see that x, y, width and height are returned.